client/state/current-user/email-verification/reducer.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import { createReducer } from 'state/utils';
+import {
+	EMAIL_VERIFY_REQUEST,
+	EMAIL_VERIFY_REQUEST_SUCCESS,
+	EMAIL_VERIFY_REQUEST_FAILURE,
+} from 'state/action-types';
+
+export const status = createReducer( null, {
+	[ EMAIL_VERIFY_REQUEST ]: () => 'requesting',
+	[ EMAIL_VERIFY_REQUEST_SUCCESS ]: () => 'sent',
+	[ EMAIL_VERIFY_REQUEST_FAILURE ]: () => 'error',
```

Comment:
Thanks! I missed that. Fixed & added a test to cover this case & also the other reducers for errorMessage (I wanted it to get reset).